### PR TITLE
Updated intro

### PR DESCRIPTION
**Pull request by JesperIRL**
I re-wrote the intro to talk more about the present and less about the past.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Reviewers
 * Magnus Ihse Bursie ([ihse](@magicus) - no project role)
 * Iris Clark ([iris](@irisclark) - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/guide pull/15/head:pull/15`
`$ git checkout pull/15`

### CODE REVIEW

File: src/intro.md [hunk 1]
```diff
@@ -4,13 +4,41 @@
...
 
-When the JDK was released under the GPLv2 license in May 2007, the need for a
-detailed document for contributors was anticipated. Translating rather
-sophisticated institutional knowledge into a format suitable for non-Oracle
-employees is always a challenge. This document attempts to address the needs of
-developers who wish to participate in the future of the JDK.
+Welcome to the OpenJDK Developers' Guide!
 
-Comments may be sent to [guide-dev (at) openjdk.java.net](mailto:guide-dev-at-openjdk.java.net).
+The OpejJDK project governs the Java programming language and the Java runtime.
```
Comment (irisclark) [inline src/intro.md:9]: Not quite.  The Java APIs, JLS, JVMS, and other normative specifications (e.g. Serialization Spec, RMI Spec, etc.) are defined by the Java Community Process (jcp.org).  The OpenJDK Project is the place to collaborate on an open-source implementation of the Java Platform, Standard Edition, and related projects.

File: src/intro.md [hunk 2]
```diff
@@ -4,13 +4,41 @@
...
+The project was created when the JDK was first released under the GPLv2 license in May
```
Comment (irisclark) [inline src/intro.md:10]: I think that "The project" should be replaced with "OpenJDK".

File: src/intro.md [hunk 3]
```diff
@@ -4,13 +4,41 @@
...
+processes are described in other documents, like [JEP 1](https://openjdk.java.net/jeps/1)
+for the JDK Enhancement-Proposal & Roadmap Process, or [JEP 3](https://openjdk.java.net/jeps/3)
+for the JDK Release Process. This guide is meant to be a complement with tutorials
+and examples for how to follow these rules and how to work together with the rest
+of the OpenJDK community.
+
+There are many common use cases that aren't detailed in the formal process. This
+guide contains the defacto standard for how to work in such cases.
+
+## Maintenance
```
Comment (magicus) [inline src/intro.md:32]: A "Maintenance" heading in the intro sounds like maintenance on OpenJDK itself.

The maintenance on the guide itself is rather un-interesting. Maybe you should add a trailing chapter "About this Guide" or something like that, where you can put stuff like that.

File: src/intro.md (hunk 1 above)
Comment (magicus) [inline src/intro.md:9]: Typo "Opej".

Comment (JesperIRL) [reply src/intro.md]: Right. I'll take that sentence :-)

File: src/intro.md (hunk 3 above)
Comment (JesperIRL) [reply src/intro.md]: Agreed. A new chapter added for the about stuff.

File: src/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 :::
 
 * [Introduction](intro.html)
-  * [Release Notes](intro.html#release-notes)
+  * [Maintenance](intro.html#maintenance)
```
Comment (magicus) [inline src/index.md:8]: Don't think this should be here anymore.

Comment (JesperIRL) [reply src/index.md]: Doh.. Fixed, and fixed again. I wrote a script to generate the index automatically at make time, that should avoid these issues in the future. A few corner cases that needs to be fixed before I push that though.

**Pull request closed without merging**